index.html

```diff
@@ -375,13 +375,64 @@ <h2>JSON Schema Specifications</h2>
           <p class="issue" data-number="133">
             Write considerations for normatively referencing JSON Schema.
           </p>
-          <p class="issue" data-number="142">
-            Write section on using different representations of JSON Schema, such as YAML.
-          </p>
-          <p class="issue" data-number="146">
-            Write section on implementation considerations, such as using <code>name</code>, <code>description</code> and
-            reserved keywords such as <code>$id</code> and <code>$schema</code>.
-          </p>
+          <section class="normative">
+            <h3>Reserved Keywords</h3>
+            <p>
+               JSON Schema specifications reserve certain keywords that hold specific meanings and
+               functions during the processing of JSON Schemas. It is crucial to avoid using conflicting
+               keys when creating JSON Schemas. The specification document for each version of JSON Schema
+               lists these reserved keywords, which can be found in the table provided above.
+
+               In the upcoming sections we list <i>some</i> keywords that possess unique significant and
+               should not be used in conflicting ways.
+            </p>
+            <p>
+               Furthermore, we will identify specific keywords that are not explicitly defined by JSON Schema
+               but are emphasized in this specification to support widespread usage.
+            </p>
+            <section>
+               <h4>$id</h4>
+               <p>
+                  Across JSON Schema specifications the <code>$id</code> keyword identifies a schema resource
+                  with its canonical [[RFC-6596]] URI. The <code>$id</code> MUST be present and its value
+                  MUST represent a valid URI-reference [[RFC-3986]].
+               </p>
+               <p>
+                  It is RECOMMENDED that the value of the <code>$id</code> property match the <code>id</code>
+                  value in the <code>credentialSchema</code> object of a <a>verifiable credential</a>, and
+                  that the value of the <code>$id</code> is a network-adderssable URL.
+               </p>
+            </section>
+            <section>
+               <h4>$schema</h4>
+               <p>
+                  Across JSON Schema specifications the <code>$schema</code> keyword identifies a JSON Schema
+                  providing the feature set for a given JSON Schema specification. This property MUST be present
+                  in each schema. For example, when constructing a schema for <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-01">Draft 2020-12</a> the value of the <code>$schema</code> identifier MUST 
```

Comment:
downref bad... we can't refer to abandoned drafts from a TR track document.

Reply:
will remove